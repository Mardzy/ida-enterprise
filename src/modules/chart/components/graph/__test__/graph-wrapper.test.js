import Graph from "../graph";

describe("The graph wrapper", () => {

  it("renders an empty shell", () => {
    const wrapper = mount(<Graph/>);

    expect(toJson(wrapper)).toMatchSnapshot();
  })

});