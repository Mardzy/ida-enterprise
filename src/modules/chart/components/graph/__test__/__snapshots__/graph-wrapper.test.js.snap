// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The graph wrapper renders an empty shell 1`] = `
.c0 {
  min-height: 7.142857142857143rem;
  min-width: auto;
}

<Graph
  chartConfig={Immutable.Map {}}
  chartId={null}
  data={null}
  graphComponent={[Function]}
  loading={false}
>
  <Styled(Component)
    attached={true}
    loading={false}
    minHeight={100}
  >
    <StyledComponent
      attached={true}
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": false,
            "lastClassName": "c0",
            "rules": Array [
              "
  min-height: ",
              [Function],
              ";
  min-width: ",
              [Function],
              ";
",
            ],
          },
          "displayName": "Styled(Component)",
          "foldedComponentIds": Array [],
          "propTypes": Object {
            "minHeight": [Function],
            "minWidth": [Function],
          },
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": [Function],
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
      loading={false}
      minHeight={100}
    >
      <Component
        attached={true}
        className="c0"
        loading={false}
        minHeight={100}
      >
        <Segment
          attached={true}
          className="c0"
          loading={false}
        >
          <div
            className="ui attached segment c0"
          >
            <graphComponent
              data={null}
            />
          </div>
        </Segment>
      </Component>
    </StyledComponent>
  </Styled(Component)>
</Graph>
`;
